CIP-????/README.md

```diff
@@ -0,0 +1,246 @@
+---
+CIP: ????
+Title: Encrypted Transaction message/comment metadata (Addendum to CIP-0020)
+Status: Proposed
+Category: Metadata
+Authors:
+    - Martin Lang <martin@martinlang.at>
+    - Ola Ahlman <ola@ahlnet.nu>
+    - Andrew Westberg <andrewwestberg@gmail.com>
+    - Adam Dean <adam@crypto2099.io>
+Implementors:
```

Comment:
(@KtorZ please correct me if I'm wrong) if some of these `Implementors` already support the encryption standard then maybe the `Status:` should be merged as `Active`?

In either case, the level of support for the newer standard should be addressed in the new `Path to Active `section` (e.g. if not supported everywhere, what's the plan to increase support)

Reply:
idk, cip1 says if there are implementors it can be set to active. i leave it up to you guys...  we have a bunch of cips ative out there on the blockchain sitting on the "proposed" status. 😆

Reply:
Either way is OK with me & I already tagged @KtorZ so let's see if he feels a particular way about it.  On the other note: we're reviewing all the CIPs now for formatting to the new standard & when we find CIPs that have gone active without being documented, we'll be correcting it in the re-review.